src/assets/translations/en/main.json

```diff
@@ -2258,6 +2258,7 @@
     "availablePools": "Available Pools",
     "addLiquidity": "Add Liquidity",
     "customSlippageDisabled": "We are unable to set a custom slippage",
+    "unsafeQuote": "This deposit has high slippage. Proceed with caution.",
```

Comment:
feels like it might be nice to show the actual slippage percentage?

Reply:
Absolutely - 34bd63acf3d721a36688d9d54c56ea7c87dd3861